bazel/repository_locations.bzl

```diff
@@ -223,14 +223,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "gRPC",
         project_desc = "gRPC C core library",
         project_url = "https://grpc.io",
-        # TODO(JimmyCYJ): Bump to release 1.27
-        # This sha on grpc:v1.25.x branch is specifically chosen to fix gRPC STS call credential options.
-        version = "d8f4928fa779f6005a7fe55a176bdb373b0f910f",
-        sha256 = "bbc8f020f4e85ec029b047fab939b8c81f3d67254b5c724e1003a2bc49ddd123",
+        version = "1.34.0",
+        sha256 = "7372a881122cd85a7224435a1d58bc5e11c88d4fb98a64b83f36f3d1c2f16d39",
         strip_prefix = "grpc-{version}",
-        urls = ["https://github.com/grpc/grpc/archive/{version}.tar.gz"],
+        urls = ["https://github.com/grpc/grpc/archive/v{version}.tar.gz"],
         use_category = ["dataplane_core", "controlplane"],
-        release_date = "2020-02-11",
+        release_date = "2020-12-01",
```

Comment:
The UTC `release_date` for this release is `2020-12-02` but we haven't landed the updated date checker which checks the release date vs. the date of the last commit in the release. This is FYI and I can clean up once the new script lands.